infra/aws/README.md

```diff
@@ -7,14 +7,24 @@ Scripts, Terraform and documentation for infrastructure that the Kubernetes comm
 - [aws-costexplorer-export](./aws-costexplorer-export/) :: an exporter to provide viewing of AWS spending
 - [terraform](./terraform/) :: contains the Terraform for provisioning various resources under different AWS accounts using Terraform
 
-## Account structure
+## Organization structure
 
 Kubernetes infrastructure is nested inside of the CNCF's AWS account
 
 ```sh
 CNCF root account
 ├── Kubernetes
+│   ├── registry.k8s.io/
+│   ├── ├── k8s-infra-aws-registry-k8s-io-admin@kubernetes.io
```

Comment:
Wouid “infra aws registry” be a good description of this account? Any extra context we can provide here?

Reply:
@sftim, thank you for your comments.
Currently it just has the literal structure but I'd like to add more documentation here.
The context of the folders says that it's for AWS so it might not be entirely needed though.